health/args.go

```diff
@@ -25,6 +25,11 @@ const (
 	// ArgRegionID provides a Warnable with the ID of a DERP server involved in the unhealthy state.
 	ArgRegionID Arg = "region-id"
 
+	// ArgRegionName provides a Warnable with the name of a DERP server involved in the unhealthy state.
+	// It is used to show a more friendly message like "the Seattle relay server failed to connect" versus
+	// "relay server 10 failed to connect".
+	ArgRegionName Arg = "region-name"
```

Comment:
If this is specifically for DERP, could we call it something like `ArgDERPRegionName` / `derp-region-name`? I know we use `region-id` above, but that means that we aren't likely to conflict with e.g. a cloud region name, if we ever include that in a health argument.

Reply:
good catch!

Reply:
done